modules/ossm-performing-revisionbased-update.adoc

```diff
@@ -34,11 +34,16 @@ The {SMProductShortName} Operator deploys a new version of the control plane alo
 $ oc get istiorevisions
 ----
 
-. Move the workloads to the new control plane by updating the `istio.io/rev` label on the application namespace or the pods to the revision name of the new control plane.
+. Move the workloads to the new control plane by updating the `istio.io/rev` label on the application namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace <namespace_name> istio.io/rev=<new_revision_name>
```

Comment:
Since the namespace would have already had the label pointing to the previous revision, we should be using overwrite flag. i.e., the command should be

```suggestion
$ oc label namespace <namespace_name> istio.io/rev=<new_revision_name> --overwrite
```